sdk/storage/azfile/internal/generated/autorest.md

```diff
@@ -307,3 +312,14 @@ directive:
       return $.
         replace(/ShareUsageBytes\s+\*int32/g, `ShareUsageBytes *int64`);
 ```
+
+### Convert StringEncoded to string type
```

Comment:
this will not quite work fully to resolve that encoded bug, but we can fix this later I think. We will need the latest generator for that.

Reply:
I tried updating the code generator to _4.0.0-preview.49_ version. But there are some issues with it. I am in discussion with @jhendrixMSFT regarding that. You can check [this commit](https://github.com/souravgupta-msft/azure-sdk-for-go/commit/639dc2663d23e3037dcffe5f78e35b66048bd0cb#diff-1967425b657f4121ac3847ecc36f26075211a0f0a8d60b95bff8b8794e062f51) for reference.

Reply:
Also the fields where the `StringEncoded` field is used, are all response types. So, this transform in my opinion is safe.
The service also returns these fields as `string` type instead of what `StringEncoded` represents.

Reply:
I have updated the code generator and added the unmarshal methods as discussed.
Will add more tests for these cases.